src/scripts/content/notion.js

```diff
@@ -16,9 +16,40 @@ togglbutton.render(
       description: getDescription
     });
 
+    const wrapper = document.createElement('div');
+    wrapper.classList.add('toggl-button-notion-wrapper');
+    wrapper.appendChild(link);
+
     const root = elem.querySelector('div:nth-child(2) > div:nth-child(1) > div:nth-child(1) > div:nth-child(3)');
     if (root) {
-      root.prepend(link);
+      root.prepend(wrapper);
+    }
+  }
+);
+
+// Button renders on main page
+togglbutton.render(
+  '.notion-topbar:not(.toggl)',
+  { observe: true },
+  function (elem) {
+    function getDescription () {
+      const descriptionElem = elem.querySelector(':scope > div > div > div:last-child');
```

Comment:
Do we need page icons/emojis in description? 🤔 

![image](https://user-images.githubusercontent.com/10184544/70602198-22522380-1bf4-11ea-8801-dbdf93fca429.png)

Reply:
thanks - I also noticed it gets cut off if the description is too long, so I've changed a better selector here